conf/messages.en

```diff
@@ -184,9 +184,9 @@ viewDepartures.table.officeOfDeparture = Office of departure
 viewDepartures.table.status = Status
 viewDepartures.table.action = Action
 viewDepartures.table.action.viewErrors        = View and fix errors
-viewDepartures.table.action.viewCancellation       = View cancellation
+viewDepartures.table.action.viewCancellation  = View cancellation
 viewDepartures.table.action.viewHistory       = View history
-viewDepartures.table.action.viewPDF           = TAD PDF ({0})
+viewDepartures.table.action.viewPDF           = TAD PDF
```

Comment:
could that key be something like viewTadPDF as there is also the TSAD which might have a separate link

Reply:
Couldn't disagree. The content for that will be updated in a separate ticket unless you want something sensible here now I will update that change.